lib/cn/csched_sp3.c

```diff
@@ -1020,19 +982,11 @@ sp3_dirty_node_locked(struct sp3 *sp, struct cn_tree_node *tn)
     }
 
     if (debug_dirty_node(sp)) {
-        slog_info(
-            SLOG_START("cn_dirty_node"),
-            SLOG_FIELD("cnid", "%lu", (ulong)tn->tn_tree->cnid),
-            SLOG_FIELD("nodeid", "%-2lu", (ulong)tn->tn_nodeid),
-            SLOG_FIELD("kvsets", "%-2lu", (ulong)nkvsets_total),
-            SLOG_FIELD("keys", "%lu", (ulong)cn_ns_keys(&tn->tn_ns)),
-            SLOG_FIELD("uniq", "%lu", (ulong)cn_ns_keys_uniq(&tn->tn_ns)),
-            SLOG_FIELD("tombs", "%lu", (ulong)cn_ns_tombs(&tn->tn_ns)),
-            SLOG_FIELD("alen", "%lu", (ulong)cn_ns_alen(&tn->tn_ns)),
-            SLOG_FIELD("clen", "%lu", (ulong)cn_ns_clen(&tn->tn_ns)),
-            SLOG_FIELD("garbage", "%u", garbage),
-            SLOG_FIELD("scatter", "%u", scatter),
-            SLOG_END);
+        log_info("cn_dirty_node cnid=%lu nodeid=%-2lu kvsets=%-2lu keys=%lu uniq=%lu tombs=%lu "
+            "alen=%lu clen=%lu garbage=%u scatter=%u", tn->tn_tree->cnid, tn->tn_nodeid,
+            nkvsets_total, cn_ns_keys(&tn->tn_ns), cn_ns_keys_uniq(&tn->tn_ns),
+            cn_ns_tombs(&tn->tn_ns), cn_ns_alen(&tn->tn_ns), cn_ns_clen(&tn->tn_ns), garbage,
+            scatter);
```

Comment:
Break up to make readable and maintainable.  Most of these slog_info() calls hard-coded the function name, but we can just drop them from the log call.